chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CognitoUserController.java

```diff
@@ -42,6 +52,59 @@ public CognitoUserController(CognitoAuthenticationManager cognitoAuthenticationM
+                    + "if they have one. Said another way, an invitation can be used to create or "
+                    + "modify CHPL user accounts." + "The correct order to call invitation requests is "
+                    + "the following: 1) /invite 2) /create or /authorize. "
+                    + "Security Restrictions: ROLE_ADMIN and ROLE_ONC can invite users to any organization.  "
+                    + "ROLE_ACB can add users to their own organization.",
+            security = {
+                    @SecurityRequirement(name = SwaggerSecurityRequirement.API_KEY),
+                    @SecurityRequirement(name = SwaggerSecurityRequirement.BEARER)
+            })
+    @RequestMapping(value = "/invite", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = "application/json; charset=utf-8")
+    public CognitoUserInvitation inviteUser(@RequestBody CognitoUserInvitation invitation) throws UserCreationException, UserRetrievalException, UserPermissionRetrievalException {
+
+        CognitoUserInvitation createdInvitiation = cognitoUserManager.inviteAdmin(invitation.getEmail());
+
+        return createdInvitiation;
+    }
+
+    @Operation(summary = "Create a new user account from an invitation.",
+            description = "An individual who has been invited to the CHPL has a special user key in their invitation email. "
+                    + "That user key along with all the information needed to create a new user's account "
+                    + "can be passed in here. The account is created but cannot be used until that user "
+                    + "confirms that their email address is valid. The correct order to call invitation requests is "
+                    + "the following: 1) /invite 2) /create or /authorize ",
+            security = {
+                    @SecurityRequirement(name = SwaggerSecurityRequirement.API_KEY)
+            })
+    @RequestMapping(value = "/create", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = "application/json; charset=utf-8")
+    public @ResponseBody BooleanResult createUser(@RequestBody CreateUserFromInvitationRequest userInfo) throws ValidationException, EmailNotSentException, UserCreationException {
+
+        Boolean success = false;
+        try {
+            //This should set the security context to user "invited user" role
+            Authentication authenticator = AuthUtil.getInvitedUserAuthenticator(null);
+            SecurityContextHolder.getContext().setAuthentication(authenticator);
+            CognitoUserInvitation invitation = cognitoUserManager.getInvitation(UUID.fromString(userInfo.getHash()));
+            if (invitation != null) {
+                success = cognitoUserManager.createUser(userInfo);
+            }
+        } finally {
+        SecurityContextHolder.getContext().setAuthentication(null);
+        }
+        return BooleanResult.builder()
```

Comment:
We do this in a few places where we return a boolean, but more recently I've been just making a void method that doesn't return anything. The response code of 200 indicates that things were successful. What do you think about doing that here?